src/cpp/rtps/builtin/BuiltinProtocols.cpp

```diff
@@ -82,7 +82,10 @@ bool BuiltinProtocols::initBuiltinProtocols(
 
     // TODO(jlbueno) The access to the list should be protected with the PDP mutex but requires a refactor on PDPClient
     // and PDPServer: read the remote servers list after the initialization of PDP.
```

Comment:
We should remove it. Note that only the participant is aware of the existence of the attributes at this stage preventing data races. The `FileWatch` callback cannot access this member on participant creation because the callback traverses the domain participant list that is not updated yet at this stage.